Tests/ClientTests/Frontend/Home/JumpBackIn/JumpBackInViewModelTests.swift

```diff
@@ -170,7 +170,8 @@ class JumpBackInViewModelTests: XCTestCase {
         trait.overridenVerticalSizeClass = .regular
 
         subject.didLoadNewData()
-        subject.refreshData(for: trait, isPortrait: true, device: .phone)
+        let size = CGSize(width: 390, height: 844)
```

Comment:
Comment so we know this size is for what device? Or maybe we could create constants with device sizes in this test class so we know implicitly from the constant name what this width and height means.